src/integration_tests/conftest.py

```diff
@@ -18,13 +19,19 @@
 
 def pytest_sessionstart():
     """Create the buckets before running the test."""
-    if os.environ.get("STORAGE_EMULATOR_HOST"):
+    if config.CONF == "docker-dev":
         try:
-            storage_client.create_bucket(os.environ.get("DATASET_BUCKET"))
+            if not gcp_storage.Bucket(
+                storage_client, config.DATASET_BUCKET_NAME
+            ).exists():
+                storage_client.create_bucket(config.DATASET_BUCKET_NAME)
         except exceptions.Conflict:
             pass
         try:
-            storage_client.create_bucket(os.environ.get("SCHEMA_BUCKET_NAME"))
+            if not gcp_storage.Bucket(
+                storage_client, config.SCHEMA_BUCKET_NAME
+            ).exists():
+                storage_client.create_bucket(config.SCHEMA_BUCKET_NAME)
```

Comment:
These try excepts are doing very similar stuff, you could refactor the logic into a function with the config value as a parameter and call it twice